test/penalizer/Utils.ts

```diff
@@ -113,13 +113,13 @@ export class RelayHelper {
     )
   }
 
-  createReceipt (relayRequest: RelayRequest, signature: string): CommitmentReceipt {
+  createReceipt (relayRequest: RelayRequest, signature: string, time: number = Math.floor(Date.now() / 1000)): CommitmentReceipt {
```

Comment:
> also, while we're at it, is the default value set to "now" so that claims are valid by default?
>
>although if this is the case, what about the slack time of 15 seconds?
>
> would appreciate your thoughts on this

Yep, ATM now is a valid value, but it actually depends on how this function is called, or better **when** it's called and on the use case. i.e. do we want to create a receipt that will be accepted? Do we want to create a receipt that will be rejected because of the time? that's why I added the possibility to specify the time 😅

Reply:
> > also, while we're at it, is the default value set to "now" so that claims are valid by default?
> > although if this is the case, what about the slack time of 15 seconds?
> > would appreciate your thoughts on this
> 
> Yep, ATM now is a valid value, but it actually depends on how this function is called, or better **when** it's called and on the use case. i.e. do we want to create a receipt that will be accepted? Do we want to create a receipt that will be rejected because of the time? that's why I added the possibility to specify the time sweat_smile

sounds good to me. it would be more confusing for this function to return a receipt which is rejected by default, instead of accepted